config/locales/en.yml

```diff
@@ -152,6 +152,20 @@ en:
           friend at %{invitation_page}.
           Cheers,
           %{archive_name}
+    admin_hidden_work_notification:
+      subject: "[%{app_name}] Your work has been hidden by the Abuse Team"
+      hello: "Dear %{user},"
+      tos: "Terms of Service"
+      contact_abuse: "contact our Abuse Team"
+      sincerely: "Sincerely,"
+      signature: "AO3 Abuse"
+      part2: "We are investigating the matter and will contact you once the investigation has been completed. While the work is under investigation it has been hidden; only you will be able to access it."
+      html:
+        part1: "Your work %{title} has been reported as being in violation of the Archive of Our Own's %{tos}."
+        part3: "If you have any questions, please %{abuse}."
+      text:
+        part1: "Your work %{title} has been been reported as being in violation of the Archive of Our Own's Terms of Service (%{tos_url})."
+        part3: "If you have any questions, please %{abuse}"
```

Comment:
I think this should be `part3: "If you have any questions, please contact our Abuse Team (%{abuse})."`. Otherwise, the email will say, "If you have any questions, please https://archiveofourown.org/abuse_reports/new."